hack/build-cross.sh

```diff
@@ -12,6 +12,11 @@ if [[ -n "${OS_ONLY_BUILD_PLATFORMS-}" ]]; then
       filtered+=("${platform}")
     fi
   done
+  if [[ ${OS_ONLY_BUILD_PLATFORMS} == "linux/ppc64le" ]]; then
```

Comment:
I still feel like appending these architectures to `$OS_CROSS_COMPILE_PLATFORMS`, the list of possible platforms, and using `$OS_ONLY_BUILD_PLATFORMS`, the platform whitelist, is the correct approach. We should look at the places we call the cross-compile scripts and make sure we are passing in the platform whitelist correctly.